### PR TITLE
build stop/start added

### PR DESCRIPTION
**Pull request by pashidlos**
https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/57

### CODE REVIEW

**Comment by pashidlos** [comment]
Code style issues to be addressed in scope of https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/94

File: visual_regression_tracker/visualRegressionTracker.py [hunk 1]
```diff
@@ -14,41 +14,49 @@ def __init__(self, config: Config):
...
-            raise Exception("Build id is not defined")
-
-        if build.projectId:
-            self.projectId = build.projectId
-        else:
-            raise Exception("Project id is not defined")
+        self.buildId = build.id
+        self.projectId = build.projectId
+
+    def stop(self):
```
Comment (dcrowe) [inline visual_regression_tracker/visualRegressionTracker.py:40]: Is it acceptable to submit additional test results after the build is stopped? Perhaps the stop function should clear the buildId and projectId.

File: README.md [hunk 2]
```diff
@@ -34,7 +37,14 @@ config = Config(
 vrt = VisualRegressionTracker(config)
 ```
 
-### Send image
+### Setup
+
+```python
+vrt.start();
```
Comment (dcrowe) [inline README.md]: This example could be made more "pythonic" if context manager support was added.
The code could become:
`with vrt:
    vrt.track(...)`

I'm happy to add the change if you would like.

**Review by dcrowe — APPROVED** [review]
I've added two minor thoughts about possible improvements, but it's definitely suitable as it stands.

File: visual_regression_tracker/visualRegressionTracker.py (hunk 1 above)
Comment (pashidlos) [reply visual_regression_tracker/visualRegressionTracker.py:40]: good point! 
let me change it

File: README.md (hunk 2 above)
Comment (pashidlos) [reply README.md]: sure, please, change it

File: visual_regression_tracker/visualRegressionTracker.py (hunk 1 above)
Comment (pashidlos) [reply visual_regression_tracker/visualRegressionTracker.py:40]: updated

**Review by dcrowe — APPROVED** [review]

**Comment by pashidlos** [comment]
@dcrowe thanks for sorting this out!  
let me try publishing it )

**Pull request merged**